src/modules/apm/sentry.exception.filter.ts

```
@@ -1,21 +1,27 @@
+        this.logger.debug(`In SentryFilter`);
+        this.logger.debug(`sending exception to sentry`, JSON.stringify(exception));
         Sentry.captureException(exception);
 
         if ((host as any).contextType && (host as any).contextType === 'graphql') {
             //do nothing as we want to see the GQL errors in the manager
+            this.logger.debug(`doing nothing so we see the errors passed along to the gql request`);
         } else {
             //do the normal error stuff, so that we return the http error codes if it was a http request
-            super.handleUnknownError(exception, host, applicationRef);
+            this.logger.debug(`calling super`);
+            return super.handleUnknownError(exception, host, applicationRef);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)